test/contract.js

```diff
@@ -218,6 +241,31 @@ describe('web3.eth.contract', function () {
 
         });
 
+        it('should explicitly make a call with optional params and defaultBlock', function () {
+           
+            var provider = new FakeHttpProvider();
+            web3.setProvider(provider);
+            web3.reset();
+            provider.injectResult('0x0000000000000000000000000000000000000000000000000000000000000032');
+            var signature = 'balance(address)';
+            var address = '0x1234567890123456789012345678901234567890';
+            provider.injectValidation(function (payload) {
+                assert.equal(payload.method, 'eth_call');
+                assert.deepEqual(payload.params, [{
+                    data: '0x' + sha3(signature).slice(0, 8) + '0000000000000000000000001234567890123456789012345678901234567890',
+                    to: address,
+                    from: address,
+                    gas: '0xc350'
+                }, '0xb']);
+            });
+
+            var contract = web3.eth.contract(desc).at(address);
+
+            var r = contract.balance.call(address, {from: address, gas: 50000}, 11);
```

Comment:
update*

ok, i see that it will work, but 

```javascript
contract.balance.call(address, {from: address, gas: 50000}, 11, function (result) { } )
contract.balance(address, {from: address, gas: 50000}, 11, function (result) { } )
```

juz doesn't look nice :/

what do you think about?

```javascript
contract.atBlock(11).balance.call(address, {from: address, gas: 50000}, function (result) { } )
contract.atBlock(11).balance(address, {from: address, gas: 50000}, function (result) { } )
```

although it's still not perfect